dev-tools.sh

```diff
@@ -106,3 +106,20 @@ dev-clear() {
 	dev clear-caches &> /dev/null &&
 	echo "All caches cleared successfully..."
 }
+
+# usage: dev-restart
+#
+# For when things are just getting a little too funky in your VM.
+# This is the last step before calling Ghost Busters.
+dev-restart() {
+	echo "Halting the VM..."
+	# Actually halt the VM.  Don't do dev stop - that only suspends the VM and you may end up with same problems once
+	# it starts back up
+	cd ~/zumba/git/onboarding &&
+	vagrant halt &&
```

Comment:
@ZurabWeb the `~/zumba/git/onboarding/vagrant halt` doesn't work for me?  that doesn't exist, and wasn't sure what you meant there.

Regarding dev stop - no, the problem with dev stop and dev start, is it does a suspend instead of actually stopping the VM.  So if you want to reboot in order to re-set-up something in system, it doesn't work.  (Like it says in the comments)